notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb

```diff
@@ -0,0 +1,2444 @@
+{
```

Comment:
<p>no 1st person, use 2nd person</p><p> </p><p>we -&gt; you</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>